services/README.md

```diff
@@ -0,0 +1,37 @@
+# Services
+
+This is a top level container for the various microservices. It allows you to mount the various endpoints at one port on one machine and makes a development vagrant/docker configuration easier to produce.
+
```

Comment:
No I thought we were trying to avoid the use of CLAW in documentation, or did I misunderstand that?